### DISCUSSION TITLE
Import CURL is so cool

### DISCUSSION THREAD

**Discussion by noam-honig** [thunderclient/thunder-client-support#723] [General]
Just wanted to say how much I love the import CURL - it's simple and works.

I can copy any request from chrome's developer tools directly to thunder client.
Simply right-click on the request, select: "copy/copy a curl bash" and then in vscode select "import curl" and I have the request right there.

Just wanted to say it's great!

**Comment by rangav** [comment]
HI @noam-honig , Glad you like the feature, Very happy to hear that. Thanks for taking time to share you feedback.

### New Features
We added 3 new features for `Curl` in `v1.17.0`, incase if you are not aware, these might be useful.

1. Now you can enter `Curl Text` or `Url` in the same input ( shortcut `Ctrl/Cmd + u` )
<img width="755" alt="Screenshot 2022-08-01 at 14 06 06" src="https://user-images.githubusercontent.com/8637550/182155125-46e13863-2001-40a1-8bb5-714cc360061d.png">

2. Added a new command `Thunder Client: Import Curl` to command palette, just search `curl`, which is helpful to test a  request from anywhere. Here also you enter Curl text or Url
<img width="740" alt="Screenshot 2022-08-01 at 14 07 47" src="https://user-images.githubusercontent.com/8637550/182155373-e0798210-8e2f-4dd7-86d1-9b643343a0f5.png">

3. When you import curl command, the request will `execute automatically` now. No need to click `Send` button

see all feature released in `v1.17.0`
https://github.com/rangav/thunder-client-support/releases/tag/v1.17.0
